source/adapter.screenshare.js

```diff
@@ -6,7 +7,35 @@ AdapterJS.TEXT.EXTENSION = {
   BUTTON_CHROME: 'Go to Chrome Web Store'
 };
 
-AdapterJS.defineMediaSourcePolyfill = function () {
+// Define extension settings
+AdapterJS.extensionInfo =  AdapterJS.extensionInfo || {
+  chrome: {
+    extensionId: 'ljckddiekopnnjoeaiofddfhgnbdoafc',
+    extensionLink: 'https://chrome.google.com/webstore/detail/skylink-webrtc-tools/ljckddiekopnnjoeaiofddfhgnbdoafc',
+    // Flag for extension versions that does not honor the "mediaSource" flag
+    legacy: true,
+    // Deprecated! Define this to use iframe method
+    iframeLink: 'https://cdn.temasys.com.sg/skylink/extensions/detectRTC.html'
+  },
+  // Required only for Firefox 51 and below
+  firefox: {
+    extensionLink: 'https://addons.mozilla.org/en-US/firefox/addon/skylink-webrtc-tools/'
+  },
+  opera: {
+    // Define the extensionId and extensionLink to integrate the Opera screensharing extension
+    extensionId: null,
+    extensionLink: null
+  }
+};
+
+AdapterJS._mediaSourcePolyfillIsDefined = false;
+AdapterJS._defineMediaSourcePolyfill = function () {
+  // Sanity checks to prevent re-defining the polyfills again in any case.
```

Comment:
@miniruwan it would make sense for pluginInfo and extensionInfo but for this case, it's something internally for us AdapterJS to use. The pluginInfo and extensionInfo is something for app developers to use, override and configure their own custom extensionId.